### DISCUSSION TITLE
Air 2S compatible with Rosetta?

### DISCUSSION THREAD

**Discussion by estevezb** [RosettaDrone/rosettadrone#177] [General]
Inspired by this group and Prof van calster, my goal is to achieve similar results with the Air 2S using AI and python scripts. Has anyone used Rosetta for Air 2S?

**Comment by titangil** [comment]
Yes, I am using it for my master thesis project. It works fine.

**Comment by kripper** [comment]
Added to README

**Comment by estevezb** [comment]
Thanks to your team. Greatly appreciate it.

**Comment by estevezb** [comment]
This is great. Thanks 🙏
